apps/_documentation/static/index.html

```diff
@@ -53,20 +53,41 @@ <h2>Table of Content</h2>
     <script src="js/markmin.js"></script>
     <script>
     var chapters = {};
-    var display = function(target, chapter) {
-         jQuery(target).markmin(chapters[chapter]);
-         Prism.highlightAll();
+    var display = function(target, chapter, anchor) {
```

Comment:
Needed to update the index.html to fix the MM links with anchor. The existing anchors weren't working, they were just displayed as [[Text_title anchor_ref]] as it was written. 
With the changes made, the links with anchor can be like:
\[Text_title\](\#chapter-xx) to refer to a chapter
\[Text_title\](\#anchor_name) to refer to an anchor in the same chapter
\[Text_title\](\#chapter-xx#anchor_name) to refer to an anchor in a different chapter